src/index.html

```diff
@@ -11,8 +11,57 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Ensure that the SCSS file is correctly compiled to CSS by the Parcel bundler, as browsers do not interpret SCSS directly.